### PR TITLE
master-lps-116337

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-116337

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase [hunk 1]
```diff
@@ -402,6 +402,175 @@ definition {
...
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "userea@liferay.com");
+
+		User.openUsersAdmin();
+
+		User.assignRegularRoleCP(
+			roleTitle = "Administrator",
+			userScreenName = "usersn");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase:425]: Something we can do here in the future is to refactor this so it's assigned through the backend. Ideally we are not doing any test setup in the UI.

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/376<br></br>

**Pull request closed without merging**